### ISSUE TITLE
coqdep cannot handle [Load string] command correctly

### ISSUE THREAD

**Issue by pi8027** [coq/coq#8273]
#### Version

8.8.1 compiled with OCaml 4.07.0

#### Operating system

Debian GNU/Linux 9.5 (stretch)

#### Description of the problem

```
$ cat x
#!/bin/sh

cat > test.v << EOT
Load "test/test.v".
EOT

coqdep test.v
$ ./x
test.vo test.glob test.v.beautified: test.v test.v test.v test.v test.v test.v test.v test.v test.v test.v ...
test.vio: test.v test.v test.v test.v test.v test.v test.v test.v test.v test.v ...
```

**Comment by pi8027** [comment]
The `dirname` part of path seems to be dropped:
https://github.com/coq/coq/blob/c66c27781c555ec7301300cbf0d0342394c03981/tools/coqdep_common.ml#L422-L423

**Comment by SkySkimmer** [comment]
Fixed in a56bda9ee4df08032bed8b9a6b4970fe9becffb5 (probably)
